packages/fluentui/docs/src/examples/components/Form/Variants/FormExampleComponents.tsx

```diff
@@ -27,8 +27,8 @@ const FormExampleComponents = () => (
   <Form>
     <Form.Input label="First name" name="firstName" id="first-name-inline" required />
     <Form.Dropdown label="City" id="cities-id" items={['prague', 'new york']} />
-    <Form.RadioGroup label="Pizza" vertical defaultCheckedValue="prosciutto" items={items} />
-    <Form.Slider />
+    <Form.RadioGroup label="Pizza" id="pizza-selection" vertical defaultCheckedValue="prosciutto" items={items} />
```

Comment:
Should `id` be required prop?

Reply:
Now I am looking deeper to it and recognized, as I change the logic for screen reader narration we would not need  id. Because this was used for `<label for='id'>`
So I could change from
` labelId.current = getOrGenerateIdFromShorthand('form-label-', id, labelId.current);`   
into 
`  labelId.current = getOrGenerateIdFromShorthand('form-label-', label, labelId.current);`

and remove all dependencies on id.

If @jurokapsiar will agree with the new reader approach then I would remove `id` prop. 
@assuncaocharles what do you think about removing id ? do we need it for some other reason like for usage in `label`  element?

Reply:
from where does `label` in `getOrGenerateIdFromShorthand('form-label-', label, labelId.current);` come from?

Reply:
sorry I didn't mention the file where we generate it, it is in file: formFieldBase.tsx

Reply:
In this case we are never going to have 
```
<label for="id" />
<input id="id" />
```

And always 

```
<label id="id-pizza">pizza</label>
<input aria-labelledby="id-pizza" />
```

Right? Sounds ok for me

Reply:
@assuncaocharles 
yes exactly. We will always narrate label with `aria-labelledby`. This should works always and consistently. :)

Reply:
I removed ID from props.